KinSDK/KinSDK/Core/blockchain/Stellar.swift

```diff
@@ -47,15 +47,15 @@ public enum Stellar {
                                    amount: Int64,
                                    memo: Memo = .MEMO_NONE,
                                    node: Node,
```

Comment:
can we rename this file/enum from Stellar? slowly transitioning to Kin blockchain instead?

Reply:
Yes, this is on my list of SDK cleanup tasks